data/synthetics_worker_versions.json

```diff
@@ -0,0 +1,6 @@
+[
```

Comment:
@hestonhoffman we may want to git ignore this file if it's going to be written to by automation.

Reply:
I think adding this to ignore would cause the build to fail, wouldn't it? Cos the partial reads that file. 

Similar to `data/sdk_versions.json`, I want to use the GitHub workflow to check for version updates and update this file for use with the build. I didn't want to incorporate this code into the build, cos it causes a potential failure if the network request fails for some reason.